### ISSUE TITLE
Suggestion: Ribbon button for customization sync & command for opening log

### ISSUE THREAD

**Issue by Ellpeck** [vrtmrz/obsidian-livesync#291]
Hi! I have two suggestions that would make using LiveSync's various UIs a little easier for people who prefer the ribbon menu, as well as for people who prefer using the command palette:

- Currently, the `Show log` button is only available in the ribbon, and not as a command. For users who don't use the ribbon, or want to keep it clean and remove the `Show log` button, there's no other way to open the log view. It'd be nice to also have `Show log` available as a command.
- I'd love it if the `Show customization sync dialog` was also part of the ribbon menu! Users who change their configuration frequently have to open the customization sync dialog a lot, and so having it easily accessible in the ribbon would be lovely.

Thanks so much for making LiveSync either way, and I hope you consider my suggestions ❤️

**Comment by Ellpeck** [comment]
Hi! I noticed that you implemented the `Show log` command, which is awesome! ❤️

I'm wondering if you're also planning on implementing my other suggestion, a `Show customization sync dialog` ribbon button? Thanks so much either  way for all your hard work, of course!!

**Comment by dlphnkck** [comment]
Hello. Please excuse me. 
You can use [the Commander plugin](https://github.com/phibr0/obsidian-commander) as one of the alternatives for your request functionality.
It is easy to use and the quickest way is to install the Commander plugin and open the options instead of reading the manual.

[phibr0/obsidian-commander: Commander - Obsidian Plugin | Add Commands to every part of Obsidian's user interface](https://github.com/phibr0/obsidian-commander)

The Commander plugin

1. any command you want, e.g. `Show log`, `Show customization sync dialog`
2. anywhere you like, e.g. Ribbon menu, Tab Bar, Status Bar, Edit Menu, File Menu, Explorer, Toolbar, etc. 
3. You can register with any icon you like.

That is all. Just for reference.
I hope this is not misplaced.

**Comment by vrtmrz** [comment]
Sorry for being absent! In recent weeks, I did something about LiveSync at midnight and have fallen asleep without writing anything. (Always, writing things makes a terrible situation, if with sleepy eyes). 

> Hi! I noticed that you implemented the Show log command, which is awesome! ❤️

I am so happy that you have noticed. I think that it may be helpful to combine some other plugins that as @dlphnkck mentioned. Thank you for introducing it to us! 
I think that it is an elegant way of Obsidian. And, in light of these extensionality combinations, I am worrying that it might be a bit intrusive to place more icons in a prime location.
If you do not mind so much, I hope to leave it at that. Would you mind if I ask your opinion, please?

**Comment by Ellpeck** [comment]
Hi! I agree that using other plugins specifically made for a task is a great idea! However, I also think that adding additional ribbon buttons doesn't necessarily increase intrusiveness per se, since hiding certain ribbon buttons is functionality that is built natively into Obsidian. If you're concerned about cluttering the ribbon menu regardless, you could always add an option to your plugin settings to disable LiveSync's ribbon buttons altogether.

I also think that, since you already have two ribbon buttons in the plugin for options that people may use frequently, it only makes sense to add the Customization Sync dialog button as well, since users will be using it frequently. 

Of course, your opinion is what counts most! Also, please take care of your health!! It sounds like you put a ton of work into LiveSync and, of course, we all appreciate it, but it sounds like you could use with taking breaks more frequently ❤️

**Comment by vrtmrz** [comment]
I appreciate your kindness! While staying up late has become a habit for me, it is still important to take care of ourselves, isn't it? I am so poor at long-drawn-out contests or marathons, and I have always forgotten that totally and every time.  However, I have realised that even if I tend to lose pace on my own, we can go far and make this better and better with you all. I feel that I am a very fortunate person and our project is blessed. Thank you so much! 

Indeed, let us return to your suggestion.

It is indeed, as you say, that just toggling is enough to control placing it. In addition, I noticed that maybe LiveSync could be the first plugin for users using hidden file sync or customisation sync. In those scenes, we probably need the ribbon buttons, which is from your idea.

This will be implemented!
